samples/Mvc.DynamicCorsPolicyProvider/corsSettings.json

```diff
@@ -0,0 +1,15 @@
+﻿[
+  {
+    "Name": "Policy1",
+    "AllowedOrigins": [
+      "http://foo.com",
```

Comment:
Try to use IANA-reserved example domains, such as `example.org`, `example.com`, and `example.net` (including all their sub-domains, e.g. `site1.example.com`).